imap_processing/cdf/utils.py

```diff
@@ -36,7 +40,9 @@ def calc_start_time(shcoarse_time: int):
 
 
 def write_cdf(
-    data: xr.Dataset, description: str = "", mode: str = "", directory: str = ""
+    data: xr.Dataset,
+    description: str = "",
+    directory: Optional[Path] = None,
```

Comment:
Should we also update the `Parameters` section of the docstring to reflect that `directory` is now a `Path` object instead of the `str`? (Looks like that whole section is not formatted correctly too)

Reply:
Good catch! I updated all of this to be the Numpy docstring style. I think we can use the type-hints to automagically generate these if we wanted to go that route as well and avoid having to manually document these...